app/scripts/services/roles.factory.js

```
@@ -2,29 +2,25 @@
  * Created by ozknemoy on 21.06.2017.
  */
 
-export default function rolesFactory() {
+export default function rolesFactoryService() {
   'ngInject';
   var roles = {};
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)